manifests/3.1.0/opensearch-3.1.0-test.yml

```diff
@@ -182,3 +182,8 @@ components:
       additional-cluster-configs:
         opensearch.experimental.feature.application_templates.enabled: true
         cluster.application_templates.enabled: true
+  - name: user-behavior-insights
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
```

Comment:
Need to add smoke-test config after https://github.com/opensearch-project/user-behavior-insights/issues/77 is closed.

Reply:
Will this PR be blocked until the smoke test is added?

Reply:
I feel like this would not block tho.
Can add in another PR if needed.

Thanks.

Reply:
Yes it can be added later. Just a reminder :)